includes/controls/groups/typography.php

```diff
@@ -88,17 +86,11 @@ public static function get_type() {
 	protected function init_fields() {
 		$fields = [];
 
-		$default_fonts = Plugin::$instance->kits_manager->get_current_settings( 'default_generic_fonts' );
-
-		if ( $default_fonts ) {
-			$default_fonts = ', ' . $default_fonts;
-		}
-
 		$fields['font_family'] = [
 			'label' => _x( 'Family', 'Typography Control', 'elementor' ),
 			'type' => Controls_Manager::FONT,
 			'default' => '',
-			'selector_value' => 'font-family: "{{VALUE}}"' . $default_fonts . ';',
+			'selector_value' => 'font-family: "{{VALUE}}", ' . Plugin::$instance->kits_manager->get_current_settings( 'default_generic_fonts' ) . ';',
```

Comment:
Since the comma shouldn't be here when there's no default font, revert the code as it was before

Reply:
@kobizz  It should be handled by the CSS Parser, something like `{{FONT_VALUE}}` that will automatically add the default font if exist.